src/databricks/labs/remorph/resources/reconcile/dashboards/recon_details_pivot.sql

```diff
@@ -0,0 +1,22 @@
+with tmp as (select recon_table_id, inserted_ts ,recon_type, explode(data) as data,
+row_number() over(partition by recon_table_id,recon_type order by recon_table_id) as rn
+from IDENTIFIER(:catalog || '.' || :schema || '.details' )
+where recon_type != 'schema')
+select main.recon_id,
+main.source_table.`catalog` as source_catalog,
+main.source_table.`schema` as source_schema,
+main.source_table.table_name as source_table_name,
+CASE
```

Comment:
Please update the `CASE..WHEN..END` statement to `if(cond, exp1, exp2)`... to simplify the statement and improve readability.

```suggestion
IF(ISNULL(MAIN.SOURCE_TABLE.CATALOG)
    , CONCAT_WS('.', MAIN.SOURCE_TABLE.SCHEMA, MAIN.SOURCE_TABLE.TABLE_NAME)
    , CONCAT_WS('.', MAIN.SOURCE_TABLE.CATALOG, MAIN.SOURCE_TABLE.SCHEMA, MAIN.SOURCE_TABLE.TABLE_NAME)) AS source_table,
```

Reply:
New files were added